Sources/Identity/CustomerInfo.swift

```diff
@@ -141,6 +141,14 @@ import Foundation
 
         let allEntitlementsDescription = self.entitlements.all.mapValues { $0.description }
 
+        let verificationResult: String
+
+        if #available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *) {
+            verificationResult = self.entitlements.verification.debugDescription
```

Comment:
Hmm I guess verificationResult is not technically part of the `customerInfo` but of `EntitlementInfos`... But I guess it's also related 1-1 currently, so I'm ok adding it to the description.